python/hsfs/ge_validation_result.py

```diff
@@ -174,10 +184,42 @@ def expectation_config(self, expectation_config):
                 "Expectation config field must be stringified json or dict"
             )
 
-    def __str__(self):
+    @property
+    def validation_time(self) -> Optional[int]:
+        return self._validation_time
+
+    @validation_time.setter
+    def validation_time(
+        self, validation_time: Union[str, int, datetime, date, None]
+    ) -> None:
```

Comment:
A doc string describing different input type of validation_time would be helpful.

Reply:
We don't want the user to set anything on these objects. They should really just be intermediate for us so I am not sure of the value but I will add something.